src/parser/warlock/affliction/modules/soulshards/SoulShardDetails.js

```diff
@@ -48,14 +48,12 @@ class SoulShardDetails extends Analyzer {
     return (
       <Statistic
         position={STATISTIC_ORDER.CORE(3)}
-        size="small"
+        size="flexible"
+        tooltip={(<>In order for Focus Magic to compete with the other talents on that row, you need to ensure you are getting as much uptime out of the buff as possible. Therefore, if you forget to put the buff on another player or if they player you gave it to is not getting crits very often, then you might need to consider giving the buff to someone else. Ideally, you should aim to trade buffs with another mage who has also taken Focus Magic so you both get the full benefit.</>)}
```

Comment:
Oof

Reply:
oops ..... Ill fix it and throw in another PR